node/test/go/dev-mode/test/fixtures/lambdas/callback.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
```

Comment:
Let's maybe name it `4s-logger` to distinguish from `callback` run in case of other packages (it'll also help when we will merge all fixtures)